docs/developer/plugin-list.asciidoc

```diff
@@ -460,6 +460,10 @@ Elastic.
 |Welcome to the Kibana rollup plugin! This plugin provides Kibana support for Elasticsearch's rollup feature. Please refer to the Elasticsearch documentation to understand rollup indices and how to create rollup jobs.
 
 
+|{kib-repo}blob/{branch}/x-pack/plugins/runtime_fields[runtimeFields]
+|WARNING: Missing README.
```

Comment:
Can we add a readme? I understand it might not contain much info at this point and is subject to change.

Reply:
I have the README.md ready in my following PR 😊